hack/istio/install-istio-via-istioctl.sh

```diff
@@ -410,12 +410,10 @@ if [ "${DELETE_ISTIO}" == "true" ]; then
   echo Deleting Core Istio
   if [ "${PURGE_UNINSTALL}" == "true" ]; then
     echo "Purging all Istio resources"
-    # Although the 'uninstall' command has been available since Istio 1.7 and is listed in the official istio doc,
-    # it's still available in 'experimental' (x) scope, so it might change in the future.
     # The optional --purge flag will remove all Istio resources, including cluster-scoped resources that may be shared with other Istio control planes.
-    ${ISTIOCTL} x uninstall --purge -y
+    ${ISTIOCTL} uninstall --purge -y
   else
-    ${ISTIOCTL} manifest generate --set profile=${CONFIG_PROFILE} ${MANIFEST_CONFIG_SETTINGS_TO_APPLY} | ${CLIENT_EXE} delete -f -
+    ${ISTIOCTL} manifest generate --set profile=${CONFIG_PROFILE} ${MANIFEST_CONFIG_SETTINGS_TO_APPLY} | ${CLIENT_EXE} delete --ignore-not-found=true -f -
```

Comment:
I think we still want the ignore-not-found... this could mean something got deleted already.  There is no need to fail this command if a resource doesn't exist - that's what we want anyway :) As long as the resources don't exist, that is all we care about (whether this command deleted it or something else did, it does not matter).

Reply:
Yes, --ignore-not-found property is specified in istio documentation (https://istio.io/latest/docs/setup/install/istioctl/#uninstall-istio) and I thought the same as you (no need to fail the command if a resources does not exist).

Reply:
Ha! I think I read that wrong. I thought you took it OUT.. lol... I'm not paying attention. I didn't realize you added that.

I approve this PR :)

Reply:
For the record, the uninstall went into "supported" mode in v1.15 of Istio:

notice the 1.14 docs vs. 1.15 docs:
* https://istio.io/v1.14/docs/setup/install/istioctl/#uninstall-istio
* https://istio.io/v1.15/docs/setup/install/istioctl/#uninstall-istio

so this uninstall will only work with 1.15 and up. Just an FYI.